design/nations/nation-validators.js

```diff
@@ -5,6 +5,24 @@ function (newDoc, oldDoc, userCtx, secObj) {
       throw { forbidden: errMessage };
     }
   }
+  function canWrite() {
+    var CAN_WRITE = true;
+    var readOnly = ['learner'];
+    if (!userCtx.isUserAdmin) {
```

Comment:
I don't think user profile properties are on a userCtx object.

Reply:
had tested it and it was working by that code.